filetracker/utils.py

```diff
@@ -62,6 +62,18 @@ def mkdir(name):
             raise
 
 
+def rmdirs(name, root):
+    """Removes empty directories from ``name`` upwards, stops at ``root``."""
+    while name != root:
+        try:
+            os.rmdir(name)
```

Comment:
Jeszcze tutaj pętla nieskończona, zaraz poprawię + testy.

Reply:
skończona, za drugim razem dostaniesz ENOENT ;)
Ale tak, teraz już powinno być ok.

Reply:
Jeśli wszystko OK, to poproszę o wyklikanie tego faktu w githubie. :)